monitoring/terraform/modules/dlq_to_slack_alerts/lambda.tf

```diff
@@ -0,0 +1,43 @@
+data "archive_file" "dlq_to_slack_alerts" {
+  type        = "zip"
+  source_file = "${path.module}/../../../dlq_to_slack_alerts/src/dlq_to_slack_alerts.py"
```

Comment:
Could we have a note on how we deploy this lambda?

Reply:
Added – the Lambda gets automatically deployed when you apply Terraform.